state-chain/chains/src/arb/api.rs

```diff
@@ -166,9 +166,26 @@ impl<E> From<EvmTransactionBuilder<transfer_fallback::TransferFallback>> for Arb
 	}
 }
 
-impl<E> RejectCall<Arbitrum> for ArbitrumApi<E> where
-	E: EvmEnvironmentProvider<Arbitrum> + ReplayProtectionProvider<Arbitrum>
+impl<E> RejectCall<Arbitrum> for ArbitrumApi<E>
+where
+	E: EvmEnvironmentProvider<Arbitrum> + ReplayProtectionProvider<Arbitrum>,
 {
+	fn new_unsigned(
+		_deposit_details: <Arbitrum as Chain>::DepositDetails,
+		refund_address: <Arbitrum as Chain>::ChainAccount,
+		refund_amount: <Arbitrum as Chain>::ChainAmount,
+		asset: <Arbitrum as Chain>::ChainAsset,
+		deposit_fetch_id: Option<<Arbitrum as Chain>::DepositFetchId>,
+	) -> Result<Self, RejectError> {
+		Ok(Self::AllBatch(evm_all_batch_builder::<Arbitrum, _>(
+			deposit_fetch_id
+				.map(|id| vec![FetchAssetParams { deposit_fetch_id: id, asset }])
+				.unwrap_or_default(),
+			vec![TransferAssetParams { asset, amount: refund_amount, to: refund_address }],
+			E::token_address,
+			E::replay_protection(E::vault_address()),
+		)?))
```

Comment:
For arbitrum we do an `AllBatch`, for ethereum we have a dedicated `RejectCall` method. Is the disparity intentional?

Reply:
I agree it should be consistent. We don't need to create a `RejectCall`, I'll remove it from Ethereum.